engine/src/main/java/org/terasology/logic/players/DebugControlSystem.java

```diff
@@ -51,8 +55,8 @@
 
     @In
     private NUIManager nuiManager;
-
-    private DebugOverlay overlay;
+    
```

Comment:
Might want to check for stray whitespace like this, since it doesn't really serve a difference other than making the diff bigger :-)